ci/test_integration_espnet2.sh

```diff
@@ -148,7 +157,14 @@ done
 rm -rf exp dump data
 cd "${cwd}"
 
+# Uninstall task-dependency
+python3 test_utils/uninstall_extra.py
```

Comment:
Can this uninstall `[task-asr]` libraries?
Why not use `pip uninstall`?

Reply:
I tried `pip uninstall espnet[task-asr]` but it will uninstall only the espnet, and not the packages in [task-asr]. I used several commands/tools to uninstall only [task-asr] related packages, and this is the best way so far. Other tools couldn't uninstall dependencies or agrgessively uninstall everything including pip.

![image](https://github.com/user-attachments/assets/98358961-94d0-4666-b48b-06357a747593)
![image](https://github.com/user-attachments/assets/763406ea-3b92-4fe8-b06d-84fc993b1c8e)
![image](https://github.com/user-attachments/assets/0595ef95-3fae-4d0c-a052-334884705a59)